### PR TITLE
Bulk broken link updates

### PR DESCRIPTION
**Pull request by mohitp930**
This PR contains fixes to broken links - [User story 168849](https://dev.azure.com/msft-skilling/Content/_workitems/edit/168849)

### CODE REVIEW

**Comment by mohitp930** [comment]
@ecfan This pull request is ready for review. Couple notes for this. There does not seem to be a Microsoft hosted download link for the WCF LOB adapter SDK, but there is a bunch of documentation related to it, so I wonder if I've just missed it somewhere in my search. Also the SAP client links in software-prerequisites-for-biztalk-adapter-pack-2016.md might be hosted behind the SAP download center, but I don't have an account to verify.

File: biztalk/adapters-and-accelerators/adapter-sap/step-2-create-an-application-definition-file-for-the-sap-artifacts.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ manager: "anneta"
 
 ## Prerequisites
 
--   Be sure that you have the Business Data Catalog Definition Editor installed as part of the Microsoft Office SharePoint Server 2007 SDK. You can download the SDK from [https://go.microsoft.com/fwlink/?LinkId=104130](https://go.microsoft.com/fwlink/?LinkId=104130).
+-   Be sure that you have the Business Data Catalog Definition Editor installed as part of the Microsoft Office SharePoint Server 2007 SDK.
```
Comment (ecfan) [inline biztalk/adapters-and-accelerators/adapter-sap/step-2-create-an-application-definition-file-for-the-sap-artifacts.md:43]: Consistency edit:

```suggestion
-   You must have the Business Data Catalog Definition Editor installed as part of the Microsoft Office SharePoint Server 2007 SDK.
```

File: biztalk/adapters-and-accelerators/adapter-oracle-ebs/step-2-create-an-application-definition-file-for-the-oracle-ebs-artifacts.md [hunk 2]
```diff
@@ -34,7 +34,7 @@ manager: "anneta"
 
 ## Prerequisites
 
--   Be sure that you have the Business Data Catalog Definition Editor installed as part of the Microsoft Office SharePoint Server 2007 SDK. You can download the SDK from [https://go.microsoft.com/fwlink/?LinkId=104130](https://go.microsoft.com/fwlink/?LinkId=104130).
+-   Be sure that you have the Business Data Catalog Definition Editor installed as part of the Microsoft Office SharePoint Server 2007 SDK.
```
Comment (ecfan) [inline biztalk/adapters-and-accelerators/adapter-oracle-ebs/step-2-create-an-application-definition-file-for-the-oracle-ebs-artifacts.md:37]:
```suggestion
-   You must have the Business Data Catalog Definition Editor installed as part of the Microsoft Office SharePoint Server 2007 SDK.
```

**Review by ecfan — CHANGES_REQUESTED** [review]
Feedback suggestions, thanks!

**Review by ecfan — APPROVED** [review]
Thanks, @mohitp930!

**Pull request merged**